doc/locale/ja/LC_MESSAGES/install/cmake_options.po

```diff
@@ -0,0 +1,161 @@
+# -*- po -*-
+# Japanese translations for Groonga package.
+# Copyright (C) 2009-2022 Groonga Project
+# This file is distributed under the same license as the Groonga package.
+# Automatically generated, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Groonga 12.0.8\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2022-09-02 03:00+0900\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid "CMake options"
+msgstr "CMake オプション"
+
+msgid "This section describes important options of CMake."
+msgstr "このセクションはCMakeの重要なオプションについて記載しています。"
+
+msgid ""
+"To get more details about installing Groonga from the source with CMake on a "
+"specific environment, find the document for the specific environment from :"
+"doc:`/install`."
+msgstr ""
+":doc:`/install` にある特定環境用のドキュメントに、CMakeでビルドする際のその環"
+"境向けのより詳細な情報があります。特定環境用のドキュメントがある場合は、まず"
+"そちらを参照してください。"
+
+msgid "``-G GENERATOR``"
+msgstr ""
+
+msgid "Specify a generator."
+msgstr "ジェネレータを指定します。"
+
+msgid "This is a mandatory option."
+msgstr "必須オプションです。"
+
+msgid "You can check available generators by ``cmake --help``."
+msgstr "``cmake --help`` で利用可能なジェネレータを確認することができます。"
+
+msgid ""
+"Here is an example of specifying ``Visual Studio 12 2013 Win64`` as a "
+"generator::"
+msgstr ""
+"以下はジェネレータに ``Visual Studio 12 2013 Win64`` を指定する例です。"
```

Comment:
```suggestion
"以下はジェネレータに ``Visual Studio 12 2013 Win64`` を指定する例です。 ::"
```

Reply:
Didn't you get a Sphinx warning without `::`?

Reply:
Hmm, I couldn't find a warning for it. I will confirm whether Sphinx puts warning again.

Reply:
Sphinx doesn't put a warning for lacking of `::` in my environment.

Reply:
Anyway, I have fixed this message.